README.md

```diff
@@ -29,14 +29,23 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. Describe the biggest difference between `.forEach` & `.map`.
+.map returns an array, .forEach does not return.
 
 2. What is the difference between a function and a method?
+A function returns a value but a method is a part of a class.
+
 
 3. What is closure?
+Closure is where an inner function has access to the outer function's variables. 
 
 4. Describe the four rules of the 'this' keyword.
```

Comment:
Window - default binding of the 'this' keyword
Implicit - method invocation binding of 'this' happens automatically with objects and methods
Explicit - uses function methods: call, apply, bind to explicitly bind 'this' to a function
New - used to create new objects and pass 'this' into constructor functions

Reply:
got it